<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RangeImage</class>
 <widget class="QWidget" name="RangeImage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>252</width>
    <height>474</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Ignored" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>244</width>
    <height>244</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 9pt &quot;微软雅黑&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>3</number>
   </property>
   <item>
    <widget class="QVTKWidget" name="qvtkWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="dspin_noise_level">
     <property name="prefix">
      <string>noise level: </string>
     </property>
     <property name="suffix">
      <string> f</string>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
     <property name="value">
      <double>0.000000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDoubleSpinBox" name="dspin_min_range">
     <property name="prefix">
      <string>min range: </string>
     </property>
     <property name="suffix">
      <string> f</string>
     </property>
     <property name="singleStep">
      <double>0.010000000000000</double>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QSpinBox" name="spin_border_size">
     <property name="suffix">
      <string/>
     </property>
     <property name="prefix">
      <string>border size: </string>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVTKWidget</class>
   <extends>QWidget</extends>
   <header>QVTKWidget.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
